packages/ibm-products/src/components/BigNumbers/BigNumbers.tsx

```diff
@@ -63,27 +60,27 @@ export let BigNumbers = React.forwardRef(
     {
       // The component props, in alphabetical order (for consistency).
       className,
-      forceShowTotal = defaults.forceShowTotal,
-      fractionDigits = defaults.fractionDigits,
+      forceShowTotal = false,
+      fractionDigits = 1,
       iconButton,
-      loading = defaults.loading,
+      loading = false,
       label,
-      locale = defaults.locale,
-      percentage = defaults.percentage,
-      size = defaults.size,
+      locale = DefaultLocale,
+      percentage = false,
+      size = 'default',
       tooltipDescription,
-      total = defaults.total,
-      trending = defaults.trending,
-      truncate = defaults.truncate,
-      value = defaults.value,
+      total,
+      trending = false,
+      truncate = true,
+      value,
       // Collect any other property values passed in.
       ...rest
-    },
-    ref
+    }: BigNumbersProps,
+    ref: React.ForwardedRef<HTMLDivElement>
   ) => {
```

Comment:
minor stylistic thing here, but i think it's cleaner to adopt this signature
```
const Component = forwardRef<HTMLDivElement, Props>((props, ref,) => {
  const { prop1 = 'default value', prop2, prop3, ...rest } = props;
  ...
});
```
this way you get more stuff out of the signature and into the body

Reply:
Normally I would not have an opinion about what is cleaner code. But I found that moving the props into the body breaks the "Default" column in Storybook.

![image](https://github.com/user-attachments/assets/96e69a53-1abf-4f34-8c6c-dc3623914ad9)

Reply:
Changed as requested, with the exception that the spread `props` were put back as a function argument.